extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -501,9 +500,9 @@ export class AmpA4A extends AMP.BaseElement {
         }
       }
       if (rendered instanceof Error) {
-        throw rendered;
+        throw dev().assertError(rendered);
```

Comment:
Ideally we wouldn't need it, but the linting system can only account for so many circumstances. The `dev().assertError()` is an easy way for the linter to ensure this is really an error instance. Plus, `dev().assertError(rendered)` turns into `rendered` when we compile the production code.